crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -177,3 +209,49 @@ pub fn sort_class_name(class_name: &TokenText, sort_config: &SortConfig) -> Stri
 
     result
 }
+
+// Returns the offset when sorting the class name.
+pub fn sort_class_name_range_offset(
+    class_name: &TokenText,
+    ignore_prefix: bool,
+    ignore_suffix: bool,
+) -> (u32, u32) {
+    let mut class_iter = class_name.split_whitespace();
+    let first_class_len = class_iter.next().map_or(0, |s| s.len()) as u32;
+    let last_class_len = class_iter.next_back().map_or(0, |s| s.len()) as u32;
+    let offset_prefix = if ignore_prefix { first_class_len } else { 0 };
+    let offset_suffix = if ignore_suffix { last_class_len } else { 0 };
+
+    (offset_prefix, offset_suffix)
+}
+
+// Returns whether the given node should be ignored prefix or suffix when sorting.
+pub fn check_ignore(node: &AnyClassStringLike) -> (bool, bool) {
```

Comment:
I would call this function `should_ignore_boundaries`, or I would create two functions called `should_ignore_prefix` and `should_ignore_postfix`.

I prefer the second option, because the name of the functions are more explicit

Reply:
For this function, is it better to return bool or Option<bool>?

Reply:
A `bool` is easier to use